dapp/src/utils/contracts.js

```diff
@@ -386,7 +386,7 @@ export async function setupContracts(account, library, chainId) {
       const response = await fetch(process.env.APR_ANALYTICS_ENDPOINT)
       if (response.ok) {
         const json = await response.json()
-        const apy = aprToApy(parseFloat(json.apr), 7)
+        const apy = aprToApy(parseFloat(json.apr), 30)
```

Comment:
As we've discussed this in the past, there is no need to do client side APY calculation since the analytics backend already returns that info